client/src/reducers/index.js

```diff
@@ -67,20 +67,12 @@ const events = (
   state = defaultEventState,
   action
 ) => {
-
-  switch(action.type) {
-
-    case FETCH_EVENTS:
-      return {
-        events: [... action.events]
-      }
-
-    // cant use ADD_EVENT here since we are using different serializers for read and write
-    // in write mode the organisation object isnt returned. So we simply fetch all again
-
-    default:
-      return state
+  if (action.events) {
+    return {
+      events: Object.values(action.events.entities.events)
```

Comment:
@QaseemLodhi why events.entities.events? I think we need to rename the first events here.